builtin/mainmenu/common.lua

```diff
@@ -93,30 +66,29 @@ function render_serverlist_row(spec, is_favorite)
 		end
 	end
 
-	local grey_out = not is_server_protocol_compat(spec.proto_min, spec.proto_max)
+	local grey_out = not spec.is_compatible
 
-	local details
-	if is_favorite then
-		details = "1,"
-	else
-		details = "0,"
-	end
+	local details = {}
 
 	if spec.ping then
 		local ping = spec.ping * 1000
 		if ping <= 50 then
-			details = details .. "2,"
+			table.insert(details, "1")
 		elseif ping <= 100 then
-			details = details .. "3,"
+			table.insert(details, "2")
 		elseif ping <= 250 then
-			details = details .. "4,"
+			table.insert(details, "3")
 		else
-			details = details .. "5,"
+			table.insert(details, "4")
 		end
 	else
-		details = details .. "0,"
+		table.insert(details, "0")
 	end
 
+	table.insert(details, "")
+	table.insert(details, "")
```

Comment:
What for?

Reply:
The dividers. I will probably attach a screenshot.